server/command.go

```diff
@@ -21,14 +21,19 @@ func (p *Plugin) createCommand(syncLinkedChannels bool) *model.Command {
 		p.API.LogWarn("Unable to get the MS Teams icon for the slash command")
 	}
 
+	autoCompleteData := getAutocompleteData(syncLinkedChannels)
+	for i := range autoCompleteData.SubCommands {
+		p.subCommands = append(p.subCommands, autoCompleteData.SubCommands[i].Trigger)
```

Comment:
To confirm, do we only ever call `createCommand ` once? Otherwise, this write to `p.subCommands` might be racy, and I wonder if we should just resolve it on demand vs trying to cache?

Reply:
We call it on onActivate but also when the configuration changes. I can add a lock tho, better be safe than sorry.